libraries/lib-import-export/ImportProgressListener.h

```diff
@@ -26,6 +28,12 @@ class IMPORT_EXPORT_API ImportProgressListener
    };
    
    virtual ~ImportProgressListener();
+
+   /// Set of functions called by importer before getting file handle as it may take a while to process
+   /// when dealing with file promise on macOS
```

Comment:
I think doxygen-style comment will be recognized as comment to the first function only, not a whole group. Should be single-line C-style comment `//`

Reply:
fixed